fd/example_test.go

```diff
@@ -21,7 +21,7 @@ func ExampleDerivative() {
 	// with a custom step size.
 	df := fd.Derivative(f, 0, &fd.Settings{
 		Formula: fd.Forward,
-		Step:    1e-8,
+		Step:    1e-6,
```

Comment:
Make this 1e-3? I didn't think about it before, but it's weird to have the "custom" step size be practically identical to the default step size.